files/chosen/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "chosen",
```

Comment:
I think you want this name to be chosen-js, `chosen` on npm looks like the wrong thing